tilewe/engine.py

```diff
@@ -33,22 +34,27 @@ def on_search(self, board: tilewe.Board, _seconds: float) -> tilewe.Move:
 
 class Tournament: 
 
-    def __init__(self, engines: list[Engine]): 
-        self.engines = list(engines) 
+    def __init__(self, engines: list[Engine], move_seconds: int=60): 
+        self.engines = list(engines)
+        self.move_seconds = move_seconds
 
-    def play(self, n_games: int, n_threads: int=1): 
+    def play(self, n_games: int, n_threads: int=1, move_seconds: int=None):
+        # initialize trackers and game controls
         games = 0
         wins = [0 for _ in range(len(self.engines))]
         totals = [0 for _ in range(len(self.engines))]
+        if move_seconds is not None: self.move_seconds = move_seconds
```

Comment:
Didn't want to have to do this necessarily, but since _play_game can only take one argument due to imap limitations the information needs to pass through the class. It seems reasonable to allow play to do this, because you may want to call subsequent tournament.play rounds without re-initializing.

Reply:
One option we could do is send all arguments as a tuple (or a dictionary) and modify `_play_game` to expect that instead. I feel like if the user wants subsequent tournament matches to use a different time control it makes more sense for them to explicitly change it at the tournament level.